cluster-logging-operator/pkg/apis/logging/v1alpha1/types.go

```diff
@@ -0,0 +1,110 @@
+package v1alpha1
+
+import (
+  metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type ClusterLoggingList struct {
+  metav1.TypeMeta   `json:",inline"`
+  metav1.ListMeta   `json:"metadata"`
+  Items             []ClusterLogging `json:"items"`
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type ClusterLogging struct {
+  metav1.TypeMeta   `json:",inline"`
+  metav1.ObjectMeta `json:"metadata"`
+  Spec              ClusterLoggingSpec   `json:"spec"`
+  Status            ClusterLoggingStatus `json:"status,omitempty"`
+}
+
+type ClusterLoggingSpec struct {
+  AllInOne          bool `json:allinone`
```

Comment:
I'm open to changing this name. The idea I'm trying to convey is that we would send `app` and `infra` logs to the same place. I want to distance us from the term `ops` as that may be a misnomer

Reply:
@jcantrill I'm not sure about making this an annotation instead. I don't forsee us allowing customers to specify their specific end points unless they were doing an external log ship. That seems to be overly complicated for the 90% use case.

Reply:
imho a view from a point of view of a pipeline would be better.
i.e. we have 2 log pipelines:
app -> logging-es
infra -> logging-es

so in yaml terms smth like:
```
appLogs:
  outputType: elasticsearch
  elasticsearchEndpoint: logging-es
infraLogs:
  outputType: elasticsearch
  elasticsearchEndpoints: logging-es
```

That way if we want logs forwarded to a different place - we simply point to a different endpoint.